.github/workflows/generate.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Setup
         uses: ./.github/actions/setup
         with:
-          install_dependencies: 'false'
+          install_dependencies: 'true'
```

Comment:
Enabled this to be able to format with `black` after generation. Prettier doesn't support Python.

Reply:
We need this to be false so the lock file sync step will work. Otherwise this will fail if the sync step has something to fix.

Let's revert the change to this line and update the workflow like this

```
      - name: Normalize poetry.lock
        run: poetry lock --no-update
      - name: Normalize package-lock.json
        run: npm install
      - name: Generate code
        run: npm run generate
      - name: Install dependencies
        run: poetry install --sync
      - name: Format
        run: make format
```

Reply:
https://github.com/seamapi/python-next/pull/6/commits/843d6bdc4fb36ad05ac6e52f05758daf9cd51305